src/videos.md

```diff
@@ -1,4 +1,17 @@
 ---
 title: Videos
 layout: _layouts/Videos.jsx
+videos:
+  - id: 1
+    header: "Header"
+    link: "dQw4w9WgXcQ"
```

Comment:
Can 'link' just be the full youtube embed link? I feel like that would be easier for them to add it